Taller152/src/paqutedos/Principal.java

```diff
@@ -4,12 +4,38 @@
  * and open the template in the editor.
  */
 package paqutedos;
-
-
+import java.util.Locale;
+import java.util.Scanner;
 /**
  *
  * @author reroes
  */
 public class Principal {
-    
-}
+    public static void main(String[] args) {
+        Scanner entrada = new Scanner(System.in);
+        entrada.useLocale(Locale.US);
+        boolean bandera = true;
+        String cadenaFinal = "";
+        while (bandera) {
+            System.out.println("Ingrese el nombre del equipo:");
+            String nombre = entrada.nextLine();
+            System.out.println("Ingrese su presupuesto anual:");
+            double presupuesto = entrada.nextDouble();
+            entrada.nextLine();
+            System.out.println("Ingrese el nombre del estadio:");
+            String estadio = entrada.nextLine();
+            System.out.println("Ingrese el numero 1 si ya no desea ingresar "
+                    + "otro equipo mas");
+            int salida = entrada.nextInt();
+            entrada.nextLine();
+            cadenaFinal = String.format("%s%s, presupuesto %.1f; estadio %s\n",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.